app/src/main/java/com/nmakademija/nmaakademija/MainActivity.java

```diff
@@ -116,13 +114,16 @@ void setCurrentFragmentProperties(int position) {
                 subtitle = R.string.news;
                 break;
             case 1:
-                subtitle = R.string.academics;
+                subtitle = R.string.schedule;
                 break;
             case 2:
-                subtitle = R.string.timer;
+                subtitle = R.string.academics;
                 break;
             case 3:
-                subtitle = R.string.schedule;
+                subtitle = R.string.timer;
+                break;
+            case 4:
+                subtitle = R.string.settings;
```

Comment:
No. But when I have some time I will investigate why Title can't be retrieved from menu items.